conductor/submitter_maya.py

```diff
@@ -172,6 +172,22 @@ def constructVerbosityArg(self, renderer):
             return "%s %s" % (varbosity_flag, verbosity_level)
 
         return ""
+    
+    def getForcedArguments(self, renderer):        
```

Comment:
Is there any mechanism to override this in case we want to render with watermarks for some reason? Didn't this come up with the skydance test?

Reply:
I believe if you do `skip_license_check` it takes precedence